### PR TITLE
DE44237: Use content font size for new html files

### PR DESCRIPTION
**Pull request by brissons**
## Relevant Rally Links

[DE44237](https://rally1.rallydev.com/#/289692574792/custom/355050439968?detail=%2Fdefect%2F603035824797&fdp=true): `d2l.Tools.Content.FontSize` is not reflecting in content

## Description

If `d2l.Tools.Content.FontSize` is enabled, we want to set the `full-page-font-size` to this value for new html files. If this config variable is not set, we revert to the previous default value of `12pt`.

## Goal/Solution

Have the starting font size for html files to be equal to `d2l.Tools.Content.FontSize` if enabled.

## Video/Screenshots

font-example

## Related PRs

- https://github.com/Brightspace/lms/pull/12481
- https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/361

## Remaining Work

- [x] Dev Testing: another developer will test this code on their machine

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/state/content-file.js [hunk 1]
```diff
@@ -61,7 +62,12 @@ export class ContentFile {
 		this.fileContent = fileContent;
 		this.fileType = contentFileEntity.getFileType();
 		this.fileHref = contentFileEntity.getFileHref();
-		this.htmlTemplatesHref = contentFileEntity.getHtmlTemplatesHref();
+
+		if (this.fileType === FILE_TYPES.html) {
+			const htmlFileEntity = new ContentHtmlFileEntity(contentFileEntity._entity, this.token, { remove: () => { } });
+			this.htmlTemplatesHref = htmlFileEntity.getHtmlTemplatesHref();
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/state/content-file.js:68]: I moved the `getHtmlTemplatesHref` to `ContentHtmlFileEntity`.

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/state/content-file.js [hunk 2]
```diff
@@ -61,7 +62,12 @@ export class ContentFile {
 		this.fileContent = fileContent;
 		this.fileType = contentFileEntity.getFileType();
 		this.fileHref = contentFileEntity.getFileHref();
-		this.htmlTemplatesHref = contentFileEntity.getHtmlTemplatesHref();
+
+		if (this.fileType === FILE_TYPES.html) {
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/state/content-file.js:66]: As a side note, when we start looking at adding new files, we are probably going to have to something similar to the [d2l-activity-content-editor-detail](https://github.com/BrightspaceHypermediaComponents/activities/blob/master/components/d2l-activity-editor/d2l-activity-content-editor/d2l-activity-content-editor-detail.js) and have every file in their own view. This file is pretty large already and only supports html files!

**Review by coffeexcode — APPROVED** [review]
LGTM! Dev tested and appears to now be respecting the config variable.

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-file-detail.js [hunk 3]
```diff
@@ -92,6 +93,8 @@ class ContentFileDetail extends SkeletonMixin(ErrorHandlingMixin(LocalizeActivit
 
 			switch (contentFileEntity.fileType) {
 				case FILE_TYPES.html:
+					this.defaultFontSize = contentFileEntity.fontSize || this.defaultFontSize;
+					this.defaultFontSize += 'pt';
```
Comment (ianlaidlaw) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-file-detail.js:97]: nit: i _BELIEVE_ you could 1 liner this:
```
this.defaultFontSize = `{ contentFileEntity.fontSize || this.defaultFontSize }pt`;
```

**Review by ianlaidlaw — DISMISSED** [review]

**Review by coffeexcode — DISMISSED** [review]

**Review by ianlaidlaw — APPROVED** [review]

**Review by ChrisKraljevicD2L — APPROVED** [review]

**Pull request merged**